Gemfile

```diff
@@ -112,7 +110,7 @@ gem "clockwork"
 gem "mini_racer", "~> 0.6.3"
 gem "nokogiri", ">= 1.10.4"
 gem "image_processing"
-gem "sprockets", "~> 4.2.0"
+gem "sprockets", "~> 4.0.0"
```

Comment:
Had to downgrade this due to rails/sprockets#749 .

Reply:
Looks like the system didn't like this downgrade very much :(

Reply:
In what way?